### DISCUSSION TITLE
Hiding Login Methods

### DISCUSSION THREAD

**Discussion by marboro92** [Web3Auth/Web3Auth#928] [Help]
I am trying to hide all login methods other than email and google in the plug and play modal. Is this possible? I can't seem to find a setting that works so far, please advise.

**Comment by Saviour1001** [comment]
While initializing the modal, you can showcase which login methods should show up by passing `modalConfig` as shown below. 
```code
await web3auth.initModal({
  modalConfig: {
    [WALLET_ADAPTERS.OPENLOGIN]: {
      label: "openlogin",
      loginMethods: {
        google: {
          name: "google login",
          logoDark: "url to your custom logo which will shown in dark mode",
        },
        facebook: {
          // it will hide the facebook option from the Web3Auth modal.
          showOnModal: false,
        },
      },
      // setting it to false will hide all social login methods from modal.
      showOnModal: true,
    },
  },
}); 
```
More details regarding customising the UI according to your needs can be found [here](https://web3auth.io/docs/sdk/web/modal/whitelabel#web3auth-modal).
